docs/deployment.md

```diff
@@ -13,52 +13,65 @@ There are orchestration templates for running Jaeger with:
   * Kubernetes: [github.com/jaegertracing/jaeger-kubernetes](https://github.com/jaegertracing/jaeger-kubernetes),
   * OpenShift: [github.com/jaegertracing/jaeger-openshift](https://github.com/jaegertracing/jaeger-openshift).
 
-## Agent
+## Storage Backend
 
-Jaeger client libraries expect **jaeger-agent** process to run locally on each host.
-The agent exposes the following ports:
+Collectors require a persistent storage backend. Cassandra 3.4+ (default) and ElasticSearch are
+primary supported storage backends. 
+=======
+Collectors require a persistent storage backend. Cassandra 3.x (default) and ElasticSearch are the
+primary supported storage backends. There is ongoing work to add support for MySQL and ScyllaDB.
 
-Port | Protocol | Function
----- | -------  | ---
-5775 | UDP      | accept zipkin.thrift over compact thrift protocol
-6831 | UDP      | accept jaeger.thrift over compact thrift protocol
-6832 | UDP      | accept jaeger.thrift over binary thrift protocol
-5778 | HTTP     | serve configs, sampling strategies
+### Cassandra
 
-It can be executed directly on the host or via Docker, as follows:
+A script is provided to initialize Cassandra keyspace and schema
+using Cassandra's interactive shell [`cqlsh`][cqlsh]
 
-```bash
-## make sure to expose only the ports you use in your deployment scenario!
-docker run \
-  --rm \
-  -p5775:5775/udp \
-  -p6831:6831/udp \
-  -p6832:6832/udp \
-  -p5778:5778/tcp \
-  jaegertracing/jaeger-agent
+```sh
+MODE=test sh ./plugin/storage/cassandra/schema/create.sh | cqlsh
 ```
+or if you don't have the source code 
+```
+docker run -e MODE=... jaegertracing/jaeger-cassandra-schema
```

Comment:
Can you replace by the actual command? I think I typed this from memory and I might be wrong :)